src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,107 @@
+import {Observable} from 'rxjs';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+
+export const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @export
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @param {string} status
+   * @returns {PersonStatus.<String>}
+   * @memberof PeopleSDKAdapter
+   */
+  getStatus(status) {
+    return Object.keys(PersonStatus).find((key) => PersonStatus[key] === status);
+  }
+
+  /**
+   * Encodes the personID into base64 format and return the UUID
+   *
+   * @param {string} ID
+   * @returns {string}
+   * @memberof PeopleSDKAdapter
+   */
+  getPersonUUID(ID) {
+    return Buffer.from(ID, 'base64')
+      .toString()
+      .split('/')[4];
+  }
+
+  async getPersonPresence(ID) {
+    try {
+      const {statusList} = await this.datasource.internal.presence.list([this.getPersonUUID(ID)]);
+
+      return this.getStatus(statusList[0].status);
+    } catch (error) {
+      throw new Error(error);
+    }
+  }
+
+  /**
+   * Fetches the person data from the sdk and returns in the shape required by adapter.
+   *
+   * @param {string} ID
+   * @returns {Room}
+   * @memberof RoomsSDKAdapter
+   */
+  async fetchPerson(ID) {
+    try {
+      const {id, emails, displayName, firstName, lastName, avatar, orgId} = await this.datasource.people.get(ID);
+
+      return {
+        ID: id,
+        emails,
+        displayName,
+        firstName,
+        lastName,
+        avatar,
+        orgID: orgId,
+        status: await this.getPersonPresence(ID),
+      };
+    } catch (error) {
+      throw new Error(error);
```

Comment:
Catching and throwing the same error seems redundant. If we're not doing any error handling, we don't really need the try/catch block.

Reply:
why are we not handling errors if we can? I'm not following.